core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/test/RuleTester.java

```diff
@@ -74,7 +74,7 @@ public RuleTester(LocalQueryRunner queryRunner)
         this.typeAnalyzer = createTestingTypeAnalyzer(queryRunner.getPlannerContext());
     }
 
-    public RuleAssert assertThat(Rule<?> rule)
+    public RuleAssert assertRule(Rule<?> rule)
```

Comment:
It's an instance method - will it actually cause a name clash?

Reply:
yeah, static import + method in a class